modules/delegates.js

```diff
@@ -769,11 +769,12 @@ Delegates.prototype.generateDelegateList = function(round, source, cb, tx) {
 				.digest();
 		}
 
-		// If the round is not an exception, cache the round.
-		if (
+		const isRoundInExceptions =
 			!Array.isArray(exceptions.ignoreDelegateListCacheForRounds) ||
-			!exceptions.ignoreDelegateListCacheForRounds.includes(round)
-		) {
+			!exceptions.ignoreDelegateListCacheForRounds.includes(round);
```

Comment:
This will not work. If you want to name the variable `isRoundInExceptions`, you either do:

```
const isRoundInExceptions =
        Array.isArray(exceptions.ignoreDelegateListCacheForRounds) &&
        exceptions.ignoreDelegateListCacheForRounds.includes(round);
```
or
```
const isRoundInExceptions =
        !(!Array.isArray(exceptions.ignoreDelegateListCacheForRounds) ||
        !exceptions.ignoreDelegateListCacheForRounds.includes(round));
```

I initially suggested the first option because I find negated OR statement confusing.

Reply:
@lsilvs I added ignoreDelegateListCacheForRounds key to default/exceptions.js and got rid of Array.isArray check. That's why I got rid of that variable completely.